doc/nep1.rst

```diff
@@ -247,6 +247,10 @@ Coding Conventions
   variables remain immutable, and gives those who read the code a better idea
   of the code's purpose.
 
+- Do not use `float` for money.
+
```

Comment:
these belong somewhere else, not in nep1.rst; maybe https://nim-lang.github.io/Nim/contributing.html#best-practices

Reply:
ok